avocado/core/app.py

```diff
@@ -17,6 +17,7 @@
 """
 
 import os
+import psutil
```

Comment:
The psutil is not an avocado dependency (It is for selftests, but not for core avocado). Would you mind using our `avocado.utils.process.kill_process_tree` instead? Both functions suffer the issue where processes created at the time of discovery are not recognized/killed and ours doesn't require any additional dependency.